File: quests/rivervale/Marshal_Anrey.pl
#Quest: Leatherfoot Raiders
#Note: All text made up until someone can capture from live.

sub EVENT_SAY {
  if($text=~/hail/i) {
    quest::say("Do I look like your best friend?!!  Well I'm not, bixie buster!!  You will address an officer with the word 'Hail' like every other deputy. Stand up straight, hunchback, or you'll be [patrolling] Rivervale till dawn!");
  }
  if($text=~/patrolling/i) {
    quest::say("That's right soldier! Until you [prove] yourself as a true Leatherfoot Raider, you will be stuck with the menial guard duty tasks.");
  }
  if($text=~/prove/i) {
    quest::say("So, you think you're up for a challenge? Alright then. To prove you are worthy of joining the Leatherfoot Squad, I need you to bring me a few things. Gather up a Polar Bear Skin, a Thick Grizzly Bear Skin, a Shark Skin, and an Alligator Skin. Do this and we can discuss your future.");
  }
}

sub EVENT_ITEM {
  if(plugin::check_handin(\%itemcount, 13761 => 1, 13756 => 1, 13075 => 1, 13749 => 1)) { #Polar Bear Skin, Thick Grizzly Bear Skin, Shark Skin, Alligator Skin
    quest::say("Good work, $name. You passed the first test. If you think you are one of us, return this cap to me along with a dagger from a Dark Elf for your true reward.");
    quest::summonitem(13941); #Leatherfoot Skullcap
    quest::exp(1000);
    quest::givecash(8,1,0,0); #Copper x 8, Silver x 1
  }
  elsif(plugin::check_handin(\%itemcount, 13941 => 1, 13942 => 1)) { #Leatherfoot Skullcap, Dragoon Dirk
    quest::say("Wonderful, $name. You have proven yourself to the Leatherfoot Squad. Take this and wear it with honor.");
    quest::summonitem(12259); #Leatherfoot Raider Skullcap
    quest::exp(5000);
    quest::givecash(7,3,2,1); #Copper x 7, Silver x 3, Gold x 2, Platinum x 1
  }
  else {
    #do all other handins first with plugin, then let it do disciplines
    plugin::try_tome_handins(\%itemcount, $class, 'Warrior');
    plugin::return_items(\%itemcount);
  }
}
#END of FILE Zone:rivervale  ID:19059 -- Marshal_Anrey

File: quests/rivervale/Marshal_Anrey.lua
--Quest: Leatherfoot Raiders

function event_say(e)
	if(e.message:findi("hail")) then
		e.self:Say("Stand at attention!! You don't look fit enough to toe-wrestle my grandma!! You cannot be in my squad!! Are you [petitioning] or are you a [visitor]?");
	elseif(e.message:findi("visitor")) then
		e.self:Say("Well, why didn't you say so?! Forgive me for hollering. Allow me to introduce myself. I am Marshal Anrey Leadladle, commander of the [Leatherfoot Raiders].");
	elseif(e.message:findi("petitioning")) then
		e.self:Say("So you want to be a [Leatherfoot Raider]? What kind of joke is this? Look at you! You're a mess! Where are you [from]?");
	elseif(e.message:findi("leatherfoot raiders")) then
		e.self:Say("You must be a visitor. The Leatherfoot Raiders are the elite force of the Guardians of the Vale. I command them. It is good to meet an outsider.");
	elseif(e.message:findi("from rivervale")) then
		e.self:Say("So you're from Rivervale?!! You must be some kind of freak. No halfling from Rivervale would look, smell and act anything like you. Well, freak, if you think you're stout enough to be a Leatherfoot Raider, you prove it!! You travel the lands and return to me a polar bear skin, a grizzly bear skin, a shark skin and an alligator skin. Then maybe, just maybe, I will let you wear the cap of a Leatherfoot Raider.");
	end
end

function event_trade(e)
	local item_lib = require("items");

	if(item_lib.check_turn_in(e.trade, {item1 = 13761,item2 = 13756,item3 = 13075,item4 = 13749})) then -- Polar Bear Skin, Thick Grizzly Bear Skin, Shark Skin, Alligator Skin
		e.self:Say("Good work, "..e.other:GetName()..". You passed the first test. If you think you are one of us, return this cap to me along with a dagger from a Dark Elf for your true reward.");
		e.other:SummonItem(13941); -- Leatherfoot Skullcap
		e.other:Ding();
		e.other:AddEXP(1000);
		e.other:GiveCash(8,1,0,0);
	elseif(item_lib.check_turn_in(e.trade, {item1 = 13941,item2 = 13942})) then -- Leatherfoot Skullcap, Dragoon Dirk
		e.self:Say("Wonderful, "..e.other:GetName()..". You have proven yourself to the Leatherfoot Squad. Take this and wear it with honor.");
		e.other:SummonItem(12259); -- Leatherfoot Raider Skullcap
		e.other:Ding();
		e.other:AddEXP(5000);
		e.other:GiveCash(7,3,2,1);
	end
	item_lib.return_items(e.self, e.other, e.trade)
end

-- END of FILE Zone:rivervale  ID:19059 -- Marshal_Anrey